### PR TITLE
Add dqsegdb

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This PR adds a recipe for `dqsegdb`.

cc: @rpfisher

### CODE REVIEW

**Comment by rpfisher** [comment]
Yes, thanks.

**Comment by rpfisher** [comment]
I seem to be unable to accept the pull request, however.

**Comment by duncanmmacleod** [comment]
@conda-forge/help-python, this recipe is ready for review, see above for @rpfisher's note on maintainership.

File: recipes/dqsegdb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f0474db90c192181e8caac0340c030ef75718c14df750cd01ac7f456c64179b0
+  patches:
+    # fix python2 bug
+    - python-print.patch
```
Comment (beckermr) [inline recipes/dqsegdb/meta.yaml]: This patch should be pushed upstream in a PR.

File: recipes/dqsegdb/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+test:
+  imports:
+    - dqsegdb
+    - dqsegdb.apicalls
+    - dqsegdb.clientutils
+    - dqsegdb.jsonhelper
+    - dqsegdb.urifunctions
+  commands:
+    # scripts are not python3-compatible as of 1.5.0
```
Comment (beckermr) [inline recipes/dqsegdb/meta.yaml]: This should be reported as well.

**Pull request merged**